frontend/src/pages/ProfileEditPage.js

```diff
@@ -0,0 +1,159 @@
+import React, { useEffect } from "react";
+import Navbar from "../component/Navbar";
+import { Field, Form } from "react-final-form";
+import { useDispatch, useSelector } from "react-redux";
+import { getUserDetails, updateUserDetails } from "../actions";
+import FileField from "../component/FileField";
+import Message from "../component/Message";
```

Comment:
Better to remove unused imports or variables.

Pls also check warnings in your logs
![image](https://user-images.githubusercontent.com/20507565/137850366-da1b97f7-329f-4a7c-8b3f-4ad02a907790.png)